beacon-chain/rpc/eth/beacon/pool.go

```diff
@@ -363,6 +366,52 @@ func (bs *Server) SubmitSignedBLSToExecutionChanges(ctx context.Context, req *et
 	return &emptypb.Empty{}, nil
 }
 
+func (bs *Server) broadcastBLSBatch(ctx context.Context, ptr *[]*ethpbalpha.SignedBLSToExecutionChange) {
+	changes := *ptr
+	limit := broadcastBLSChangesRateLimit
+	if len(changes) < broadcastBLSChangesRateLimit {
+		limit = len(changes)
+	}
+	st, err := bs.ChainInfoFetcher.HeadState(ctx)
+	if err != nil {
+		log.WithError(err).Error("could not get head state")
+		return
+	}
+	for _, ch := range changes[:limit] {
+		if ch != nil {
+			_, err := blocks.ValidateBLSToExecutionChange(st, ch)
+			if err != nil {
+				log.WithError(err).Error("could not validate BLS to execution change")
+				continue
+			}
```

Comment:
Why aren't we validating the signature here?

Reply:
Discussed offline. Signature was validated earlier and this check is for reorg resistance